### ISSUE TITLE
Add pagination to a few components

### ISSUE THREAD

**Issue by awolfe76** [cfpb/hmda-platform-ui#274]
Depends on https://github.com/cfpb/hmda-platform/issues/719

Once the endpoints are returning what's needed we'll have to add pagination to:
- [ ] [`editsTable`](https://github.com/cfpb/hmda-platform-ui/blob/master/src/js/containers/EditsTable.jsx) _(on hold)_
- [ ] parserErrors (depends on #273)
- [ ] [`IRSReport`](https://github.com/cfpb/hmda-platform-ui/blob/master/src/js/components/IRSReport.jsx)_(on hold)_

We'll have to update the api calls to allow for pagination, adding `page=x`.

**Note** - macro will not need pagination. There are only 33 rows possible, so we should just show them all.

**Comment by awolfe76** [comment]
@wpears - https://github.com/cfpb/hmda-platform/pull/821 was merged so we can start this one now, at least for the parse errors.
